.appveyor.yml

```diff
@@ -1,80 +1,54 @@
-version: 0.0.1-branch-{branch}-build-{build}
+version: 2.3.18-branch-{branch}-build-{build}
```

Comment:
Do we have to hard code the version number here? :(
That's not too much of a benefit, right?

Since we would have to update that on every release, next to the version number update in the cmake file... Can't appveyor pull it from there?

Reply:
It's only used to display in AppVeyor so it's not a big deal

Reply:
Especially then we should question it!
Nobody will take the effort - especially you Zach when you do a release - to update that number, just for some displaying purpose in one of the CI's...

And if we know beforehand that it will show a wrong/outdated number sooner than later, it might be better to just scrape it right away.
Results in better and a more helpful information displayed, and less confusing for all viewer.

If you look at appveyor, there are other issues with that version tag in general as is, too...